docs/source/get-started/configuration-guide.rst

```diff
@@ -529,6 +529,10 @@ If cross-compiling, or if you want to be specific, the CPU architecture can be p
       - Milk-V Pioneer
       - (since Kokkos 4.3)
 
+    * - ``Kokkos_ARCH_RISCV_U74MC``
+      - U74MC/RISC-V ISA
+      - SiFive Unmatched
+      - (since Kokkos 4.7)
```

Comment:
Did you mean to commit these changes?

Reply:
Yes, these are committee. I forgot to merge or rebase with main. I will do that back at my computer.